model_configs/imputation/three_layer_performer.yml

```diff
@@ -0,0 +1,16 @@
+name: "PytorchImpute"
+optimizer_name: "Adam"
+loss_name: "MSELoss"
+model_name: "ImputePerformer"
+layer_count: 3
+num_heads: 8
+dim: 8
+lr: .001
```

Comment:
Do you think these hyperparameters will need to be tuned? Or are these models generally fairly robust to hyperparameter settings?

Reply:
They'll probably need to be tuned some day, but I'm putting that off until I have a working model and dataset :)